app/controllers/course_templates_controller.rb

```diff
@@ -52,7 +52,9 @@ def prepare_course
                          title: @course_template.title,
                          description: @course_template.description,
                          material_url: @course_template.material_url,
-                         source_url: @course_template.source_url
+                         source_url: @course_template.source_url,
+                         course_template_id: @course_template.id,
+                         cache_version: @course_template.cache_version
```

Comment:
maybe have also the last line ending with `,` so it would be easier to reorder/add new lines